tests/integration/test_create_token.py

```diff
@@ -0,0 +1,134 @@
+from app.authentication.authenticator import decrypt_token
+from tests.integration.app_context_test_case import AppContextTestCase
+from tests.integration.create_token import (
+    PAYLOAD_V2_BUSINESS,
+    PAYLOAD_V2_SOCIAL,
+    PAYLOAD_V2_SUPPLEMENTARY_DATA,
+)
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestCreateToken(IntegrationTestCase, AppContextTestCase):
+    """
+    The purpose of this test class is to test the creation of a token (from create_token.py) to ensure
+    metadata in a decrypted token is nested/found in the correct level.
+    """
+
+    def test_payload_from_token(self):
```

Comment:
Could be test payload structure from token or something more descriptive?

Reply:
Changed to a more suitable name 👍